src/routes/safe/components/Apps/confirmTransactions.tsx

```diff
@@ -56,16 +55,11 @@ const IconText = styled.div`
   }
 `
 const isTxValid = (t): boolean => {
-  try {
-    if (!['string', 'number'].includes(typeof t.value)) {
-      return false
-    }
-
-    if (typeof t.value === 'string') {
-      const web3 = getWeb3()
-      web3.eth.abi.decodeParameter('uint256', t.value)
-    }
-  } catch (error) {
+  if (!['string', 'number'].includes(typeof t.value)) {
+    return false
+  }
+
+  if (typeof t.value === 'string' && !/^\d+$/.test(t.value)) {
```

Comment:
What about hexadecimals?

Reply:
@rmeissner Could you please give us some insight what type of value should be valid? So far we only thought about something like '180000000000000'

Reply:
it kind of depends on you, but please be explicit about it (e.g. document it). In Ethereum normally you can specify numbers as decimals strings or hex strings. If it is a hex string we probably need to convert for the backend, so it might be fair if we limit it to strings, would still expect that this is commented. 

Also on a different note ... definitely something where I would expect a tests here (especially if this was considered a bug)

Reply:
so, we should allow numbers and strings, but strings as decimal numbers not as hexadecimal. 

Then, we should document it in the SDK Readme.

is it ok @rmeissner ?

Reply:
yes

Reply:
so @mikheevm we are good to go. the current condition works as expected

Reply:
> Also on a different note ... definitely something where I would expect a tests here (especially if this was considered a bug)

are we? :P

Reply:
if it is not a bug why are we changing it?

Reply:
we already have a ticket to adding tests to safe-apps integration. We need this fix for the coming release, can we merge it and work on all the test later?

Reply:
Mikhail found this problem https://github.com/gnosis/safe-react/issues/1032

Reply:
ok